src/components/side_panel/chart/gauge_chart_panel/gauge_chart_design_panel.ts

```diff
@@ -139,6 +145,12 @@ export class GaugeChartDesignPanel extends Component<Props, SpreadsheetChildEnv>
   }
 
   updateSectionRule(sectionRule: SectionRule) {
+    const invalidValueReasons = this.checkSectionRuleFormulasAreValid(this.state.sectionRule);
+    if (invalidValueReasons.length > 0) {
+      this.state.sectionRuleDispatchResult = new DispatchResult(invalidValueReasons);
+      return;
```

Comment:
this prevents the user from changing the general config onece there is an evaluated value taht does not fit the rule.. I think it should rather pipe the results together because I don't want to have to change the content of a cell or of my input juste to have the possibility to change the color of a threshold.